src/main/java/seedu/address/logic/commands/FindNricCommand.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.NricMatchesKeywordsPredicate;
+
+// The current format: find_nric S1234567A
```

Comment:
Hmm, we probably have to discuss this tomorrow during our weekly meeting. Do we want to separate the commands for pairs, elderlies, and volunteers? Or have a one-fits-all approach? 

The issue with the one-fits-all approach is how do we determine which Nric belongs to which Entity?

Reply:
Personally, i think it's okay to have separate commands, and do the clean up later

Reply:
We have been beating around the bush on this for quite awhile. Can get the team lead to iron this out tomorrow